samples/excel/89-preview-apis/events-tablecollection-changed.yaml

```diff
@@ -0,0 +1,164 @@
+order: 9
+id: excel-events-tablecollection-changed
+name: Events - Table collection changed
+description: Add event handlers for table collection onDataChanged event
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#register-on-data-changed-handler").click(() => tryCatch(registerOnDataChangedHandler));
+        $("#change-data").click(() => tryCatch(changeData));
+
+        async function registerOnDataChangedHandler() {
+            await Excel.run(async (context) => {
+                let table = context.workbook.tables;
+                table.onDataChanged.add(onDataChange);
+
+                await context.sync();
+
+                OfficeHelpers.UI.notify("A handler has been registered for the table collection onDataChanged event",
+                    "Try changing cell values in the tables, and watch the console output.");
+            });
+        }
+
+        async function changeData() {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getItem("Sample");
+                let range1 = sheet.getRange("B7");
+                let range2 = sheet.getRange("C15");
```

Comment:
Is there a reason why you are changing two ranges instead of one? If so, could you explain in a code comment?